### DISCUSSION TITLE
`gh pr list` Output Not Displayed Inline in Terminal

### DISCUSSION THREAD

**Discussion by lemtoc** [ohmyzsh/ohmyzsh#12737] [Help] [answered]
Hi, Thank you for the wonderful framework.

## Description:
I'm using zsh with macOS, and I've installed Oh-My-Zsh. When I run the `gh pr list` command to display the list of pull requests, the output is shown in a separate "paged" view, and I have to press `q` to exit and return to the shell. I want to display the output inline in the terminal without switching to this paged view.
With the default zsh, it is output inline, so I want to use it with the same specifications.
I'm using `gh pr list` as an example, but I don't think this is limited to that.

### Steps to Reproduce:
1. Using macOS with zsh and Oh-My-Zsh installed.
2. Run the command: `gh pr list`.
### Current Behavior:
- The output is shown in a separate view (as if opened in a less pager).
- I need to press `q` to return to the terminal prompt.
### Expected Behavior:
- I would like the pull request list to be displayed directly in the terminal without opening the pager.
#### with `oh-my-zsh`

![demo](https://github.com/user-attachments/assets/40ed9264-7809-46b4-bb96-263fac7a9769)

#### with `default zsh`

![demo](https://github.com/user-attachments/assets/2350f684-392c-4486-96d7-0edd023b5a32)

**Comment by carlosala** [comment, marked as answer]
That's an `gh` related issue, not `omz`. Nevertheless, if you use `GH_PAGER= gh pr list` it'll work out 👍🏻 
https://github.com/cli/cli/issues/3617

**Reply by lemtoc** [reply]
Hi, @carlosala 
Thank you! I was hoping for this.
I was able to achieve it by adding the following to `.zshrc`, based on your advice.
```shell
export GH_PAGER="less -RFX"
```
